rotkehlchen/chain/ethereum/tokens.py

```diff
@@ -59,8 +62,23 @@
 # tokens the benchmark will probably have to run again.
 
 
-ETHERSCAN_MAX_TOKEN_CHUNK_LENGTH = 120
-OTHER_MAX_TOKEN_CHUNK_LENGTH = 590
+ETHERSCAN_MAX_TOKEN_CHUNK_LENGTH = 121
+OTHER_MAX_TOKEN_CHUNK_LENGTH = 591
```

Comment:
why +1?

Reply:
Previously query was built in the following way:
`account` + `array of tokens` where chunk length is the length of the array of tokens.
So the total size was `..._CHUNK_LENGTH` + 1.

Now query is built in the following way:
`array of accounts` + `array of tokens` where chunk length is the sum of lengths of these arrays